ow_plexil/src/plans/HealthMonitorDemo.plp

```diff
@@ -31,37 +41,45 @@ HealthMonitorDemo: Concurrence
 
   log_info("Starting HealthMonitorDemo...");
 
-  Monitor:
-  {
-    LibraryCall HealthMonitor (AllOperable = AllGood,
-                               ArmOperable = ArmGood,
-                               AntennaOperable = AntennaGood,
-                               CameraOperable = CameraGood,
-                               PowerOperable = PowerGood);
-  }
-
-  DemoPattern:
+  Actions: Concurrence
   {
-    Repeat true;
-    Wait 3;
+    Exit Lookup(PowerFault);
```

Comment:
This PR should be removing all uses of ExitConditions, right?

Reply:
No.  Just when the node also has a Start condition.